### DISCUSSION TITLE
Embed Public GitHub Files

### DISCUSSION THREAD

**Discussion by wise-io** [GitbookIO/community#20] [Feature Requests]
### What's your feature/idea?

I'd like to be able to embed files from a public GitHub repo, similar to how embedding a public Gist works.

### What's your desired outcome?

**Example GitBook:** https://scripts.redletter.tech
**Example Public GitHub Repo:** https://github.com/wise-io/scripts
**Example Gists:** https://gist.github.com/wise-io

Currently, I have to maintain 2 copies of each file I'd like to embed: 
- One copy in my public repo (see above)
- One copy in my personal Gists for embedding in my GitBook

If we could embed files straight from a public GitHub repo, it would remove the burden of maintaining two copies of each file I'd like to embed.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by SamyPesse** [comment]
It's definitely something we'd like to build as part of an advanced GitHub integration.
We can't share an ETA at the moment, but we'll keep you posted through this discussion once it's shipped.

**Comment by wise-io** [comment]
Just wanted to say the new GitHub Files application solves this for me! The only thing missing now is code/syntax highlighting when using GitHub Files links instead of a code block. I'd love to see this added!
